src/crypto/random-browser.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const Buffer = require('safe-buffer').Buffer
+
+function getWebCrypto () {
+  if (typeof window !== 'undefined') {
```

Comment:
there were some changes to the original to support webworkers. I am not a big fan of duplicating this here, the two ways I see is either put it into its own module or require libp2p-crypto from this module. What do you think?